### DISCUSSION TITLE
Discord - Privacy Enhanced Mode Questions

### DISCUSSION THREAD

**Discussion by TFWol** [sandboxie-plus/Sandboxie#1739] [Q&A Feedback] [answered]
Hi, I'm wondering if anyone has been able to get Discord working consistently in Privacy Enhanced mode.

I'm pretty much shooting for the app to think it's the only thing installed; I'm not a fan of it trying to enumerate my system and had further kept it from seeing the 'Program Files' directories. I only install it inside sandboxes.

It was working mostly fine in 1.0.11 and earlier. In the past versions I'd have to reinstall it every time Sandboxie was updated, which was doable. 
That method won't work anymore since it only runs about 3 times before it permanently breaks with `SBIE2310 Name buffer is approaching overflow (-1)` and gets the updater stuck at '**Starting...**'

Often, I have to attempt installs by wiping out the sandbox 5-6 times (to get it working those 3 times). Also, Sandboxie will lock a file handle at System level and only a reboot would unlock it for me to delete the sandbox files.

I can't really find much in the forums/docs dealing with this feature.
So anyone have a setup/method they use in Privacy Enhanced mode that actually works?

**Comment by DavidXanatos** [comment, marked as answer]
this is strange, you sure that it broke exactyl between 1.0.11 and 1.0.12?
I wil test that so you install discord directly to the sandbox in privacy mode blue icon, not hardened privacy mode red icon?

**Reply by TFWol** [reply]
Thanks for the reply. Yeah, it definitely worked in at least version 1.0.11, but I'm not sure which version broke it since I had waited a few versions to update due to how my stuff breaks and I have to reinstall the sandboxed applications.

I install in privacy mode (blue icon).

I can reproduce the starting up issue on a test machine with nothing else installed. After 2 or 3 successful runs, Discord just won't start.

**Reply by DavidXanatos** [reply]
Ok... could you than test with all builds > 1.0.11 to find out which one exactly broke it, that would help me a lot.

**Reply by TFWol** [reply]
lol, I actually started doing that as soon as I replied. In progress.

**Reply by TFWol** [reply]
Hmm...it looks like I may have found a way forward.
The version of discord is the same (v 1.0.904), but once I changed the Run menu command line from `<truncated>..AppData\Local\Discord\app-1.0.9004\Discord.exe` to `<truncated>..AppData\Local\Discord\Update.exe --processStart Discord.exe` everything just started working.

This is SB+ V1.0.15.

I'll update Sandboxie to v1.0.16 and verify.

**Reply by TFWol** [reply]
Yep, it seems to be working fine. Kind of odd since the resulting application that gets ran is the same in the end.
I guess this can be marked as resolved.
